app/src/main/java/com/github/sdp/tarjetakuna/database/UserRTDB.kt

```diff
@@ -0,0 +1,139 @@
+package com.github.sdp.tarjetakuna.database
+
+import com.google.firebase.auth.ktx.auth
+import com.google.firebase.database.DataSnapshot
+import com.google.firebase.database.ktx.database
+import com.google.firebase.ktx.Firebase
+import java.util.concurrent.CompletableFuture
+
+/**
+ * This class is used to manage the user's data in the database.
+ */
+class UserRTDB {
+    //todo: switch to use auth adapter?
+    private val db = Firebase.database.reference
+    private val currentUser = Firebase.auth.currentUser

```

Comment:
Here as well maybe we should use the GoogleAuth to have another level of abstraction and make the tests easier